library/f5bigip_ltm_profile_web_acceleration.py

```diff
@@ -0,0 +1,174 @@
+        default: enabled
+        choices: ['disabled', 'enabled']
+    cache_max_age:
+        description:
+            - Specifies how long the system considers the cached content to be valid.
+        default: 3600
+    cache_max_entries:
+        description:
+            - Specifies the maximum number of entries that can be in the WA cache.
+        default: 10000
+    cache_object_max_size:
+        description:
+            - Specifies the largest object that the system considers eligible for caching.
+        default: 50000
+    cache_object_min_size:
+        description:
+            - Specifies the smallest object that the system considers eligible forcaching.
+        default: 500
+    cache_size:
+        description:
+            - Specifies the maximum size, in megabytes, for the WA cache.
+        default: 100
+    cache_uri_exclude:
+        description:
+            - Configures a list of Uniform Resource Identifiers (URIs) to exclude fromthe WA Cache.
+    cache_uri_include:
+        description:
+            - Configures a list of URIs that are cacheable.
+    cache_uri_include_override:
+        description:
+            - Configures a list of URIs that should be cached in the WA cache eventhough they would normally not be cached due to constraints defined byweb-accelerationTMSH Reference 67 - 203cache-object-max-size or others.
+    cache_uri_pinned:
+        description:
+            - Configures a list of URIs that are kept in the WA cache regardless theirmax-age or expiry settings.
+    defaults_from:
+        description:
+            - Configures the profile that you want to use as the parent profile.
+        default: webacceleration
+    metadata_cache_max_size:
+        description:
+            - Specifies the maximum size of the metadata cache.
+    name:
+        description:
+            - Specifies a unique name for the component.
+        required: true
+    partition:
+        description:
+            - Displays the administrative partition within which the profile resides.
+    state:
+        description:
+            - Specifies the state of the component on the BIG-IP system.
+        default: present
+        choices: ['absent', 'present']
+notes:
+    - Requires BIG-IP software version >= 11.6
+requirements:
+    - ansible-common-f5
+    - f5-sdk
+'''
+
+EXAMPLES = '''
+- name: Create LTM Monitor Web Acceleration
+  f5bigip_ltm_profile_web_acceleration:
+    f5_hostname: 172.16.227.35
+    f5_username: admin
+    f5_password: admin
+    f5_port: 443
+    partition: Common
+    name: my_web_acceleration_profile
+    state: present
+  delegate_to: localhost
+'''
+
+RETURN = '''
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_common_f5.f5_bigip import *
+
+BIGIP_LTM_PROFILE_WEB_ACCELERATION_ARGS = dict(
+    app_service=dict(type='str'),
+    applications=dict(type='list'),
+    cache_aging_rate=dict(type='int', choices=range(0,10)),
```

Comment:
Use Six library for range.

`from six.moves import range`

Reply:
It is fixed! Thanks